resources/variants/strateo3d_haute_temperature_04.inst.cfg

```diff
@@ -0,0 +1,20 @@
+[general]
+name = Haute température 0.4
```

Comment:
We require profile names to have an English name, if the name is descriptive. The Strateo3D website calls it ["high temp"](https://en.emotion-tech.com/product/1234569352-full-high-temp-hotend-for-strateo3d-0-4mm-nozzle). Could you make that consistent?

I can make this change myself too, but it would be good to have your consent. In any case I'm trying to get this profile update in for the next Cura release.

Note that this would then also need to be changed in other references to this nozzle (the quality and material profiles), because for some reason we chose to make that refer to the user-visible name instead of the nozzle ID in the distant past...

Reply:
Ok, you can make this change on your own (like strateo3d_hight_temp_04.inst.cfg), but can you also do it for the other references (quality and material profiles) ? 
Or should we change it ourselves?

Reply:
I can do this for you, yeah, no problem.